src/main/webapp/app/exam/participate/exercises/exercise-overview-page/exam-exercise-overview-page.component.html

```diff
@@ -55,12 +55,27 @@ <h2 style="text-align: left; font-weight: normal; margin-bottom: 30px">
                         {{ item.exercise.maxPoints }}
                     </td>
                     <td style="max-width: 50px; font-weight: bold">
```

Comment:
Could you try to make this a little wider? The change looks good, it just leads to situations like these with a line break in the cell:

![grafik](https://github.com/ls1intum/Artemis/assets/38403547/fad9cd45-71d7-488f-bc4b-a940170d103a)

![grafik](https://github.com/ls1intum/Artemis/assets/38403547/8b2b4d64-4129-417c-9512-3cc616bbabd6)



If it helps: the "points" column seems a bit large in comparison. Its header `<th>` doesn't appear to have a max-width.